tests/layer/test_gcnsupervisedgraphclassification.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018-2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from stellargraph.layer.graph_classification import *
+from stellargraph.mapper import GraphGenerator, FullBatchNodeGenerator
+import pytest
+from ..test_utils.graphs import example_graph_random
+
+
+class Test_GCNSupervisedGraphConvolution:
+
+    graphs = [
+        example_graph_random(feature_size=4, n_nodes=6),
+        example_graph_random(feature_size=4, n_nodes=5),
+        example_graph_random(feature_size=4, n_nodes=3),
+    ]
+
+    generator = GraphGenerator(graphs=graphs)
+
+    def test_init(self):
+
+        model = GCNSupervisedGraphClassification(
+            layer_sizes=[16], activations=["relu"], generator=self.generator
+        )
+
+        assert len(model.layer_sizes) == 1
+        assert len(model.activations) == 1
+        assert model.layer_sizes[0] == 16
+        assert model.activations[0] == "relu"
+
+        with pytest.raises(TypeError):
```

Comment:
It's not just good for reading and understanding, but it also helps ensure the test continues to check the right thing in future. If there's changes that modify the order in which things are checked or other changes to what's expected of parameters (e.g. `layer_sizes` switches to needing a tuple instead of a list).